sass/sass_wrapper.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ * 
+ * A Sass compiler wrapper that supports bazel persistent worker protocol.
+ *
+ * Bazel can spawn a persistent worker process that handles multiple invocations.
+ * It can also be invoked with an argument file to run once and exit.
+ */
+"use strict";
+const {debug, runAsWorker, runWorkerLoop} = require('@bazel/worker');
+const sass = require('sass');
+const fs = require('fs');
+
+function main(args) {
+  if (runAsWorker(args)) {
+    debug('Starting Sass compiler persistent worker...');
+    runWorkerLoop(args => sass.run_(args));
+    // Note: intentionally don't process.exit() here, because runWorkerLoop
+    // is waiting for async callbacks from node.
+  } else {
+    debug('Running a single build...');
+    if (args.length === 0) throw new Error('Not enough arguments');
```

Comment:
Why does this throw an error, where below it prints an error and returns with exit code 0?

Reply:
Changed the later statement to throw as well.